### PR TITLE
docs: Added long description to PyPi

### PR DESCRIPTION
**Pull request by benjats07**
This PR add long description to the package.

The description is taked from README.md

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,14 +17,22 @@
...
+cwd = os.path.dirname(os.path.abspath(__file__))
+
+# Read in README.md for our long_description
+with open(os.path.join(cwd, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
 setup(
     name="unstructured",
     description="A library that prepares raw documents for downstream ML tasks.",
+    long_description=long_description,
```
Comment (MthwRobinson) [inline setup.py:35]: We should be able to do this without importing `os`, as seen in `transformers` [here](https://github.com/huggingface/transformers/blob/main/setup.py#L416-L417). While we're in here, let's also add `keywords`, `url`, `python_requires`, and `classifiers`.  For `classifiers` we can borrow what the classifiers from `transformers`, except drop `Programming Language :: Python :: 3.7` and change development status to `Development Status :: 4 - Beta`

**Review by MthwRobinson — CHANGES_REQUESTED** [review]
Check out the inline comment.

File: setup.py (hunk 1 above)
Comment (benjats07) [reply setup.py:35]: @MthwRobinson  What keywords you suggest? I think maybe: 
"NLP PDF HTML parsing preprocessing"

Comment (MthwRobinson) [reply setup.py:35]: I'd add `CV` and `XML`, the others look good to me!

**Review by MthwRobinson — APPROVED** [review]
LGTM, nice work! Can you do the same in the `setup.py` for `unstructured-api-tools`?

**Comment by benjats07** [comment]
Yes, already done.

**Pull request merged**